pages/linux/gnome-software.md

```diff
@@ -0,0 +1,20 @@
+# gnome-software
+
+> Add and remove applications and update your system.
+> More information: <https://apps.gnome.org/app/org.gnome.Software/>.
+
+- Launch the GNOME Software GUI, does nothing if it's already open:
```

Comment:
```suggestion
- Launch the GNOME Software GUI, if it's not already running.
```

Reply:
```suggestion
- Launch the GNOME Software GUI if it's not already running:
```

I don't think the comma needs to be there if it's written like this. (Since the 2nd part is dependent on the 1st.)
Also, I assume period (.) is a mistake?

Reply:
I think the disclaimer is not required. A lot of apps use a single instance.

Reply:
From the perspective of a reader that interprets everything literally, I think it's a crucial detail in my opinion.

However, I'll leave it up to everyone else to decide what you think suits the repo more.

Reply:
For now, I'll commit the altered suggestion. If others also state they think the heads-up is redundant, then I'll change it up.

Reply:
Huh… didn't know this.

Even though I technically merged my suggestion, not CleanMachines, GitHub automatically adds the `Co-authored by:` thing anyway.

Pretty nice to know.